src/lib.rs

```diff
@@ -467,12 +495,8 @@ pub mod __macro_internals {
 ``` */
 ///
 #[macro_export]
-macro_rules! attribute_alias {(
-    $(
-        #[apply($name:ident $(!)?)] = $( #[$($attrs:tt)*] )+;
-    )*
-) => (
-    $(
+macro_rules! attribute_alias {
+    (#[apply($name:ident $(!)?)] = $( #[$($attrs:tt)*] )+;) => {
```

Comment:
Ditto here (also, whitespace is very important for macro readability, let's avoid squashing things together)